PsfLauncher/main.cpp

```diff
@@ -9,16 +9,20 @@
 #include <sstream>
 
 #include <windows.h>
-#include <psf_constants.h>
-#include <psf_runtime.h>
 #include <shellapi.h>
 #include <combaseapi.h>
 #include <ppltasks.h>
 #include <ShObjIdl.h>
-#include <wil\resource.h>
-#include <wil\result.h>
-#include <TraceLoggingProvider.h>
+#include "StartProcessHelper.h"
 #include "Telemetry.h"
+#include "PsfPowershellScriptRunner.h"
+#include <TraceLoggingProvider.h>
+#include <psf_constants.h>
+#include <psf_runtime.h>
+#include <future>
+#include <wil\result.h>
+#include <wil\resource.h>
+
 
 TRACELOGGING_DECLARE_PROVIDER(g_Log_ETW_ComponentProvider);

```

Comment:
>TRACELOGGING_DECLARE_PROVIDER [](start = 0, length = 29)

nit: Extra newline

Reply:
Fixed.

---
In reply to: [316806215](https://github.com/microsoft/MSIX-PackageSupportFramework/pull/79#discussion_r316806215) [](ancestors = 316806215)